test/packetimpact/testbench/layers.go

```diff
@@ -440,7 +446,7 @@ func (l *TCP) length() int {
 
 // merge overrides the values in l with the values from other but only in fields
 // where the value is not nil.
-func (l *TCP) merge(other TCP) error {
+func (l *TCP) Mergelayer(other TCP) error {
```

Comment:
I don't think it needs layer in the name because that is just repeating the type of the argument.  `Merge` is enough.

Reply:
[GES] Ok. Will remove Mergelayer(..).